python/tests/util/mark.py

```diff
@@ -45,6 +49,17 @@
 Currently controlled by the ARCTICDB_PERSISTENT_STORAGE_TESTS and ARCTICDB_FAST_TESTS_ONLY env vars."""
 
 
+@contextmanager
+def random_seed_context():
```

Comment:
Is `mark.py` really the right file to hold this function? It isn't anything to do with marks